zbus/src/object_server/interface/mod.rs

```diff
@@ -45,10 +48,19 @@ pub trait Interface: Any + Send + Sync {
     }
 
     /// Get a property value. Returns `None` if the property doesn't exist.
```

Comment:
Perhaps it'd be good to document here briefly when the `header` is `Some` and when it's `None`.